src/noise_interfaces/noise_process_interface.jl

```diff
@@ -347,7 +351,7 @@ end
     return out1,out2
   else # Bridge
     i = searchsortedfirst(W.t,t)
-    if t == W.t[i]
+    if isapprox(t, W.t[i]; rtol = 1e-8)
```

Comment:
this should probably bee something like `100eps(t)`

Reply:
but also careful of zero

Reply:
awesome. Wanted to ask you about this anyway. How should we handle the case close to 0.0?
` eps(0.0) = 5.0e-324` doesn't seems to be appropriate. Should this include some atol then as well?

Reply:
yeah, do an `atol` based on `100eps(typeof(t))`.

Reply:
are those updated ones fine?